### PR TITLE
Added Requirements.txt

### PR DESCRIPTION
**Pull request by NEERAJAP2001**
Solves #22 as of now!

### CODE REVIEW

**Comment by NEERAJAP2001** [comment]
@yvkrishna review, please.

**Comment by chebroluharika** [comment]
@NEERAJAP2001 I am not sure about few things. Please make me understand them.
1. There are cv2, os, shutil, sklearn, numpy, matplotlib dependencies listed in README, but when I see in requirements.txt file there are 3 libraries listed. Can I know the reason please?
2. From the description in issue specified #22 , I understood that you want to install dependencies in one go. But here you just listed dependencies in requirements.txt file, how will that help in resolving that issue unless you run 
`pip install requirements.txt` . If its so, please mention that in README file and update PR.
And do please let me know if I am wrong anywhere.

**Review by chebroluharika — COMMENTED** [review]
There are cv2, os, shutil, sklearn, numpy, matplotlib dependencies listed in README, but when I see in requirements.txt file there are 3 libraries listed. Can I know the reason please?
From the description in issue specified #22 , I understood that you want to install dependencies in one go. But here you just listed dependencies in requirements.txt file, how will that help in resolving that issue unless you run
pip install requirements.txt . If its so, please mention that in README file and update PR.
And do please let me know if I am wrong anywhere.

**Comment by chebroluharika** [comment]
@yvkrishna  Please add relevant labels to this PR as I don't have access to do it.

**Comment by NEERAJAP2001** [comment]
@chebroluharika These are the only dependencies, So actually there is a library [Here](https://pypi.org/project/pipreqs/) 
When I ran the command on This Repo it created the requirement.txt.
So I feel if you just run ```pip install requirements.txt``` all the dependencies are loaded up and ready to start using Flow2ML


Yeah, I just wanted confirmation for updating the ReadMe file.

**Comment by chebroluharika** [comment]
@NEERAJAP2001 Please go ahead and mention necessary steps precisely in Readme according to your analysis and update PR. I will review it then.

**Comment by NEERAJAP2001** [comment]
@chebroluharika Can you check now?

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+numpy==1.19.2
+matplotlib==3.3.4
+scikit_learn==0.24.1
+opencv==4.5.1
```
Comment (chebroluharika) [inline requirements.txt:4]: Please include os, shutil also as they are mentioned as dependencies in README.

Comment (NEERAJAP2001) [reply requirements.txt:4]: Actually, it's inbuilt in python I believe(cause I tried on my PC).
I just have python installed and the ```import shutil``` works fine.
can you please confirm?
Same for os
pip install requirements.txt

Comment (chebroluharika) [reply requirements.txt:4]: yes, if we write import statement at the top of program, it works fine as expected. 
Purpose of having requirements file was though we don't mention required packages inside program it should work fine.
Hence, please mention in requirements file too.

Comment (NEERAJAP2001) [reply requirements.txt:4]: Okay !! I'll do it now

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
```
Comment (chebroluharika) [inline requirements.txt:4]: I think this should be opencv-python because there is no opencv package to get installed with pip.

Comment (NEERAJAP2001) [reply requirements.txt:4]: Yeah, Okay

**Review by chebroluharika — APPROVED** [review]

**Comment by NEERAJAP2001** [comment]
any more @chebroluharika?

**Pull request merged**